src/olympia/activity/utils.py

```diff
@@ -0,0 +1,210 @@
+import base64
+import datetime
+import logging
+import quopri
+import re
+import urllib2
+from email import message_from_string
+from email.utils import parseaddr
+
+from django.conf import settings
+from django.template import Context, loader
+
+from email_reply_parser import EmailReplyParser
+
+from olympia import amo
+from olympia.access import acl
+from olympia.activity.models import ActivityLogToken
+from olympia.amo.helpers import absolutify
+from olympia.amo.urlresolvers import reverse
+from olympia.amo.utils import send_mail
+from olympia.devhub.models import ActivityLog
+
+log = logging.getLogger('z.amo.activity')
+
+# Prefix of the reply to address in devcomm emails.
+REPLY_TO_PREFIX = 'reviewreply+'
+
+
+class ActivityEmailParser(object):
+    """Utility to parse email replies."""
+    address_prefix = REPLY_TO_PREFIX
+
+    def __init__(self, email_text):
+        """Decode base64 email and turn it into a Django email object."""
+        try:
+            email_text = base64.standard_b64decode(
+                urllib2.unquote(email_text.rstrip()))
+        except TypeError:
+            # Corrupt or invalid base 64.
+            self.decode_error = True
+            log.info('Decoding error for ActivityEmailParser')
+            return
```

Comment:
I lifted this Parser class directly from commbadge in Marketplace and I agree it's habit of just silently dropping out with `False` or `None` is less than desirable.  I was going to add email replies to the next iteration that notifies the email sender if there was something wrong (and what).  For now I'll add exceptions as suggested.